api/src/main/java/io/minio/messages/PrefixConverter.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * MinIO Java SDK for Amazon S3 Compatible Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.minio.messages;
+
+import org.simpleframework.xml.convert.Converter;
+import org.simpleframework.xml.stream.InputNode;
+import org.simpleframework.xml.stream.OutputNode;
+
+/**
+ * XML converter class for prefix due to SimpleXML limitation in converting empty tag
+ * <Prefix></Prefix> to empty string.
+ */
+public class PrefixConverter implements Converter<String> {
```

Comment:
Since the logic is not very specific to `prefix` as such and can be useful wherever such a functionality is required (de-serialize empty tag to empty string), maybe the name of this class can be something more generic? Say `StringConverter` ?

Reply:
Currently `prefix` is only supported empty. Surely change when I find another case.